recipes/ocl-icd/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "ocl-icd" %}
+{% set version = "2.2.9" %}
+{% set sha256 = "0c8ac13e2c5b737c34de49f9aca6cad3c4d33dd0bbb149b01238d76e798feae5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://forge.imag.fr/frs/download.php/716/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+  # Makes ocl-icd install the OpenCL headers it ships
+  - install-headers.patch
+  - install-headers-post-automake.patch
+  - define-cl2-prefix-suffix-on-apple.patch
```

Comment:
No, I wrote them. I chose the names to be descriptive, so a one-liner description wouldn't necessarily add value. But I'm happy to explain what I did.